### ISSUE TITLE
Group tweets in threads into one entry

### ISSUE THREAD

**Issue by aaadelnrrv** [kickscondor/fraidycat#37]
I follow several twitter users who post long threads on various topics. It would be a nice improvement if all the tweets in a thread were combined into a single entry under the user, instead of each tweet being it's own entry so a single thread fills the entire list of links.

**Comment by kickscondor** [comment]
Mmmmn - I like this. Thank you for the suggestion!

**Comment by kickscondor** [comment]
Ok - 1.1.0 will at least filter out thread replies. And retweets - because I don't like that they are shown as if the author said them.

However, I'm leaving this open and reassigning it to 1.2.0. I'm hoping to address layout better and it would be nice to go a bit further with this:

* Show how many parts a thread has.
* Update the thread's timestamp if a reply is added.
* Show a retweet's original author.

**Comment by kickscondor** [comment]
Ok, actually fit this one into 1.1.0. Out soon...

Note that with retweets, it'll use the date of the original tweet for now.

**Comment by rootkea** [comment]
So I'm following the local municipality Twitter accounts for COVID-19 lockdown related orders. (shop timings etc.)

Everything was going okay. But one day I happen to look at the the said account on Twitter.com and I suddenly realized that I had missed couple of important updates as they were mentioned in a Twitter thread and fraidyCat only shows the first tweet of the thread without any indicator of it being part of a twitter thread.

I can understand the overwhelming when each and every tweet of the thread is displayed. But completely hiding the tweets is as equally worst.

My suggestion would be to have a little indicator (e.g. smal triangle or arrow) besides the head tweet which on clicked will expand and show the complete Twitter thread. This will make sure that the user doesn't miss any tweet.

PS - Only indicator won't do. As in that case though the user would come to know that the tweet is part of thread but then he/she will have to visit Twitter.com to read complete thread which defeats the purpose of FraidyCat. So not just an indicator but fetching/expanding the thread when that indicator is clicked is needed too.

Thanks!

**Comment by kickscondor** [comment]
I completely agree with you that this needs to be improved. I appreciate you taking the time to write out your thoughts in such detail - I'll reopen this and try to figure out how this needs to work. Thank you, Avinash!

**Comment by medicinecal** [comment]
Ok work on this is going forward. Updated threads will be treated like updated posts. This fits into the 2.0 redesign.

**Comment by rootkea** [comment]
Eagerly waiting for the update so that I can start using fraidycat again! Thank you for working on this!
